ActindoWorkFlow/UserInterface/06a_Logs.md

```diff
@@ -85,11 +46,14 @@ The list displays all logs that are matching the selected log level. Depending o
     Description of the issue.
 
 - *Workflow*   
-    Workflow which the log message refers to.
+    Workflow to which the log message refers.
 
 - *Workflow ID*       
     Workflow identification number. The ID number is automatically assigned by the system.
 
+- *Search string*   
+    String that can be used to search for a log. In most cases, the column is empty.
```

Comment:
In most cases, the column is empty. --> Ich würde es evtl. weglassen. Es klingt nicht wirklich positiv. Wozu ist die Spalte denn da, wenn fast immer leer? Und ist eigentlich die Suchfunktion nicht eine Freitext-Suche? In der vorigen Version war ein ID / RemoteId. Evtl. Entwicklung fragen?